content/hi/containerization.md

```diff
@@ -0,0 +1,17 @@
+---
+title: कंटेनरीकरण
+status: Completed
+category: प्रौद्योगिकी
+---
+
+## यह क्या है 
+
+कंटेनरीकरण एक एप्लिकेशन और उसकी निर्भरता को एक [कंटेनर इमेज](/container-image/) में बंडल करने की प्रक्रिया है। कंटेनर निर्माण प्रक्रिया के लिए [ओपन कंटेनर इनिशिएटिव](https://opencontainers.org) (OCI) मानक का पालन करना आवश्यक है। जब तक आउटपुट एक कंटेनर इमेज है जो इस मानक का पालन करती है, तब तक कोई फर्क नहीं पड़ता कि किस कंटेनरीकरण उपकरण का उपयोग किया जाता है।
+
+## समस्या  
+
+कंटेनरों के प्रचलित होने से पहले, संगठन एक ही [बेयर-मेटल मशीन](/bare_metal_machine/) पर कई अनुप्रयोगों को व्यवस्थित करने के लिए वर्चुअल मशीन (VMs) पर निर्भर थे। VMs कंटेनरों से काफी बड़े होते हैं और उन्हें चलाने के लिए एक हाइपरवाइजर की आवश्यकता होती है। इन बड़े VM टेम्प्लेट के स्टोरेज, बैकअप और ट्रांसफर के कारण, VM टेम्पलेट्स बनाना एक धीमी प्रक्रिया है। इसके अतिरिक्त, VMs कॉन्फ़िगरेशन बहाव से पीड़ित हो सकते हैं जो [अपरिवर्तनीयता](/immutable_infrastructure/) के सिद्धांत का उल्लंघन करता है।
```

Comment:
```suggestion
कंटेनरों के प्रचलित होने से पहले, संगठन एक ही [बेयर-मेटल मशीन](/bare_metal_machine/) पर कई एप्लीकेशन को व्यवस्थित करने के लिए वर्चुअल मशीनों (VMs) पर निर्भर थे। VMs कंटेनरों से काफी बड़े होते हैं और उन्हें चलाने के लिए एक हाइपरवाइजर की आवश्यकता होती है। इन बड़े VM टेम्प्लेट के स्टोरेज, बैकअप और ट्रांसफर के कारण, VM टेम्पलेट्स बनाना एक धीमी प्रक्रिया है। इसके अतिरिक्त, VMs कॉन्फ़िगरेशन ड्रिफ्ट से पीड़ित हो सकते हैं जो [अपरिवर्तनीयता](/immutable_infrastructure/) के सिद्धांत का उल्लंघन करता है।
```

Reply:
I think we should keep "configuration drift" as "कॉन्फ़िगरेशन ड्रिफ्ट" as its a commonly user phrase in industry referring to drastic changes on containers over time due to manual configuration of each container.

Reply:
+1 to @anubha-v-ardhan 's one.

Reply:
Oh, guess i missed `अनुप्रयोगों` here! 
Thanks for pointing out